addons/mail_bot/models/res_users.py

```diff
@@ -25,13 +26,14 @@ class ResUsers(models.Model):
     def SELF_READABLE_FIELDS(self):
         return super().SELF_READABLE_FIELDS + ['odoobot_state']
 
-    def _init_messaging(self, store):
-        odoobot_onboarding = False
-        if self.odoobot_state in [False, 'not_initialized'] and self._is_internal():
-            odoobot_onboarding = True
+    def _on_webclient_bootstrap(self):
+        if self and self._is_internal() and self.odoobot_state in [False, 'not_initialized']:
             self._init_odoobot()
-        super()._init_messaging(store)
-        store.add({"odoobotOnboarding": odoobot_onboarding})
+
+    def _init_store_data(self, store: Store, /):
```

Comment:
`_init_store_data` should eventually be changed to also use "self" rather than env user, but this can be another PR.